ztp/ztp-policy-generator/kustomize/plugin/policyGenerator/v1/policygenerator/hack/common.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+pushd "$(dirname "$0")/.."
+
+function finish {
+    popd
+}
+trap finish EXIT
+
+GOPATH="${GOPATH:-~/go}"
+# not yet, but soon...
+#export GOFLAGS="${GOFLAGS:-"-mod=vendor"}"
```

Comment:
@nishant-parekh : this should be uncommented once your vendor PR goes in, I think.

Reply:
I think this can be uncommented now.

Reply:
Exactly why I put the hold on it :)